android/app/src/main/java/com/example/clarity/sdk/Requests.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.clarity.sdk
+
+data class LoginRequest(val username: String, val password: String)
+data class CreateUserEntity(val user: User)
+data class User(val username: String, val email: String, val password: String, val firstname: String, val lastname: String, val phone_number: String)
+data class JoinClassroomEntity(val privateCode: String, val userID: String)
+data class CreateClassroomEntity(val name: String, val teacher: Integer)
+data class CreateCardSetEntity(val creator_id: Int, val title: String, val type: String)
+data class GetDataForSetRequest(val set_id: Int)
+data class GetDataForSetResponse(val response: StatusResponse, val data: List<String>)
```

Comment:
should be moved to Responses.kt

Reply:
same thing with:
LoginResponse
JoinClassroomResponse
GetClassroomResponse

Reply:
also wondering why LoginResponse has a (val success: Boolean) instead of a (val response: StatusResponse), was it just set up that way in the backend